source/testnet/see-also/release-notes.rst

```diff
@@ -1,13 +1,10 @@
-.. _download page: /testnet/docs/downloads
-.. _iOS Concordium ID: /testnet/docs/downloads
-.. _Added import to app: /testnet/docs/quickstart-export-import
-.. _Encrypted(shielded) amounts and transfers: /testnet/docs/quickstart-shielded-transfers
-.. _here: https://developer.concordium.com/testnet/docs/downloads#concordium-id
-.. _upgrade: /testnet/docs/quickstart-node#upgrade-from-open-testnet-v1
-.. _Concordium Node version 0.2.13: /testnet/docs/downloads#concordium-node-and-client
-.. _Concordium ID: /testnet/docs/downloads#concordium-id
-.. _Concordium Node and Client Software: /testnet/docs/downloads#concordium-node-and-client
+.. _download page: /testnet/see-also/downloads.html
+.. _iOS Concordium ID: /testnet/see-also/downloads.html
+.. _here: https://developer.concordium.com/testnet/docs/downloads.html#concordium-id
+.. _Concordium ID: /testnet/see-also/downloads.html#concordium-id
+.. _Concordium Node and Client Software: /testnet/see-also/downloads.html#concordium-node-and-client
```

Comment:
I think all of the above links could be internal.

Reply:
True. I'll fix that before merging